<?xml version="1.0"?>
<launch>
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find alliance_test)/config/rosconsole.conf"/>
  <include file="$(find alliance_test)/launch/single_robot.launch" >
    <arg name="robot_id" value="robot1" />
  </include>
  <include file="$(find alliance_test)/launch/single_robot.launch">
    <arg name="robot_id" value="robot2" />
  </include>
  <include file="$(find alliance_test)/launch/single_robot.launch">
    <arg name="robot_id" value="robot3" />
  </include>
 <include file="$(find alliance_test)/launch/single_robot.launch">
    <arg name="robot_id" value="robot4" />
  </include>
  <include file="$(find alliance_test)/launch/single_robot.launch">
    <arg name="robot_id" value="robot5" />
  </include>
  <node name="motivation_bag" pkg="rosbag" type="record"
        args="-O $(find alliance_test)/bag/motivation.bag /alliance/motivation" />
</launch>
